microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.adoc

```diff
@@ -6,5 +6,142 @@ include::_attributes/common-attributes.adoc[]
+
+LVMS uses `StorageCapacity` tracking to ensure that pods with an LVMS PVC are not scheduled if the requested storage is greater than the volume group's remaining free storage. For more information about `StorageCapacity` tracking, see link:https://kubernetes.io/docs/concepts/storage/storage-capacity/[Storage Capacity]. 
+
+[id="lvms-configuring"]
+== Configuring the LVMS 
+
+{product-title} supports passing through a user's LVMS configuration and allows users to specify custom volume groups, thin volume provisioning parameters, and reserved unallocated volume group space. The LVMS configuration file can be edited at any time. You must restart {product-title} to deploy configuration changes. 
+
+The following `config.yaml` file shows a basic LVMS configuration: 
+
+.LVMS YAML configuration 
+[source,yaml]
+----
+socket-name: <1>
+device-classes: <2>
+  - name: <3>
+    volume-group: <4>
+    spare-gb: <5>
+    default: <6>
+  - name: hdd
+    volume-group: hdd-vg
+    spare-gb: 10
+  - name: striped
+    volume-group: multi-pv-vg
+    spare-gb: 10
+    stripe: <7>
+    stripe-size: <8>
+  - name: raid
+    volume-group: raid-vg
+    lvcreate-options: <9>
+      - --type=raid1
+----
+<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  `/run/topolvm/lvmd.sock`. 
+<2> `map[string]DeviceClass`. The `device-class` settings. 
+<3> String. The name of the `device-class`.
+<4> String. The group where the `device-class` creates the logical volumes. 
+<5> unit64. Storage capacity in GiB to be spared. Defaults to `10`. 
+<6> Boolean. Indicates that the `device-class` is used by default. Defaults to `false`. 
+<7> unit. The number of stripes in the logical volume.
+<8> String. The amount of data that is written to one device before moving to the next device. 
+<9> String. Extra arguments to pas `lvcreate`, for example, `[--type=raid1"`]. 
++
+[NOTE]
+====
+Striping can be configured by using the dedicated options (`stripe` and `stripe-size`) and `lvcreate-options`. Either option can be used, but they cannot be used together. Using `stripe` and `stripe-size` with `lvcreate-options` leads to duplicate arguments to `lvcreate`. You should never set `lvcreate-options: ["--stripes=n"]` and `stripe: n` at the same time. You can, however, use both, when `lvcreate-options` is not used for striping. For example: 
+
+[source,yaml]
+----
+stripe: 2
+lvcreate-options: ["--mirrors=1"]
+----
+====
+
+[id="setting-lvms-path"]
+=== Setting the LVMS path 
+
+The `config.yaml` file for the LMVS should be written to the same directory as the MicroShift `config.yaml` file. If a MicroShift `config.yaml` file does not exist, MicroShift will create an LVMS `config.yaml` file and automatically populate the configuration fields with the default settings. The following paths are checked for the `config.yaml` file, depending on which user runs MicroShift: 
```

Comment:
Not meaning to provide technical review, but if the LVMS `config.yaml` file and the MicroShift `config.yaml` file are written to the same directory, won't one filename overwrite the other?

Or should they be in the same parent directory?

Reply:
Or do the files have a different filename? I see `lvmd.yaml` below. So maybe it should be something like, "The LMVS configuration file, `lvmd.yaml`, ...